crates/libs/com/src/lib.rs

```diff
@@ -11,9 +11,9 @@ pub mod ServiceFabric;
 // expose mod directly
 pub use ServiceFabric::*;
 
-#[cfg(target_os = "windows")]
+#[cfg(all(target_os = "windows", feature = "bundled_import_libs"))]
 pub use mssf_metadata;
 
 // In linux force to pull in pal lib for linking
-#[cfg(target_os = "linux")]
+#[cfg(all(target_os = "linux", feature = "bundled_import_libs"))]
```

Comment:
Why pal needs to have bundled_import_libs feature?
linux does not have import libs.